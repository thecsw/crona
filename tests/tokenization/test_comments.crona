// this is a comment
// this is a comment with a bad escape \k