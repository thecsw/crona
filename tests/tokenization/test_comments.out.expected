EOF [2,42]
